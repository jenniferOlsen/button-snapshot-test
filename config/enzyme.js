// enzyme setup file that configures appropriate adapter for React version
import { configure } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

configure({ adapter: new Adapter() })
