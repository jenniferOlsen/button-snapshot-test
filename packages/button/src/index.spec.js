/* global describe it expect */

import React from 'react'
import { mount } from 'enzyme'
import 'jest-styled-components'

import { Button } from '.'

describe('Button Component', () => {
  it('renders correct button styles', () => {
    const wrapper = mount(<Button />)
    expect(wrapper).toMatchSnapshot()
    expect(wrapper).toHaveStyleRule('color', 'white')
  })
})
