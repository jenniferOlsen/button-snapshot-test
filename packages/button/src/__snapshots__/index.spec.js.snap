// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Component renders correct button styles 1`] = `
<Button>
  <src__StyledButton>
    <button
      className="src__StyledButton-jTdqxW jQYvmM"
    />
  </src__StyledButton>
</Button>
`;
